core/base/signedDistanceField/SignedDistanceField.h

```diff
@@ -0,0 +1,720 @@
+/// \ingroup base
+/// \class ttk::SignedDistanceField
+/// \author Julien Tierny <julien.tierny@lip6.fr>
+/// \author Mohamed Amine Kissi <mohamed.kissi@lip6.fr>
+/// \author Mathieu Pont <mathieu.pont@lip6.fr>
```

Comment:
Can you please add a brief description of each backend? (possibly with references to the corresponding publications?) thanks!

Reply:
Done in the next commit!